docs/privacy-and-security/cloud-hardening.md

```diff
@@ -111,33 +113,33 @@ flowchart TB
 
 ## Security Hardening Areas
 
-| Section | Purpose | Risk Level |
-|--------|---------|------------|
-| **1. Secure Agent-to-Cloud Connection** | Enforce encrypted streaming and access control policies between Agents/Parents and Cloud. | **Critical** |
-| **2. Authentication and SSO** | Use identity providers and enable SSO or MFA to restrict access. | **Critical** |
-| **3. Role-Based Access Control (RBAC)** | Assign least-privilege roles to team members within Netdata Cloud Rooms. | **High** |
-| **4. Cloud Configuration Management** | Review change history for dashboards, alerts, and spaces. | **Medium** |
-| **5. Alert Notification Security** | Secure all alert endpoints and notification channels. | **High** |
-| **6. External Access Protection** | Secure the Cloud UI with SSO, enforce session expiration policies. | **Critical** |
-| **7. Integration Security** | Secure third-party endpoints (e.g., Slack, PagerDuty) with proper tokens. | **Medium** |
-| **8. Metadata Privacy** | Limit what metadata is collected by Agents and uploaded to Cloud. | **Medium** |
-| **9. Telemetry and Analytics Opt-Out** | Disable usage telemetry if needed for compliance. | **Low** |
-| **10. Compliance Mapping** | Align usage of Netdata Cloud with frameworks such as SOC 2, ISO 27001, GDPR, etc. | **Medium** |
+| Section                                 | Purpose                                                                                                       | Risk Level   |
+| --------------------------------------- | ------------------------------------------------------------------------------------------------------------- | ------------ |
+| **1. Secure Agent-to-Cloud Connection** | Enforce encrypted streaming and access control policies between Agents/Parents and Cloud.                     | **Critical** |
+| **2. Authentication and SSO**           | Use identity providers and enable SSO or MFA to restrict access.                                              | **Critical** |
+| **3. Role-Based Access Control (RBAC)** | Assign least-privilege roles to team members within Netdata Cloud Rooms.                                      | **High**     |
+| **4. Cloud Configuration Management**   | Review change history for dashboards, alerts, and spaces.                                                     | **Medium**   |
+| **5. Alert Notification Security**      | Secure all alert endpoints and notification channels.                                                         | **High**     |
+| **6. External Access Protection**       | Secure the Cloud UI with SSO, enforce session expiration policies.                                            | **Critical** |
```

Comment:
The session expiration is something I have small issue with because cloud itself does not offer such setting. This needs to be fixed on SSO provider side.

Reply:
https://github.com/netdata/cloud-backend/issues/749